code/__DEFINES/is_helpers.dm

```diff
@@ -46,6 +46,8 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 //Mobs
 #define isliving(A) (istype(A, /mob/living))
 
+#define issimp(A) (istype(A, /mob/living/simple_animal))
```

Comment:
don't make me be the killer of good times shaps

Reply:
man i thought that'd last longer than 11 minutes

Reply:
i'm sorry shaps

Reply:
this must stay in the codebase

Reply:
@Jared-Fogle ok boomer